devices/pine64-pinephone/u-boot/default.nix

```diff
@@ -58,29 +58,13 @@ in
     # https://patchwork.ozlabs.org/patch/1202024
     (pw "1202024" "0c196zk1s3pq3wdv909sxmjgqpll2hwb817bpbghkfkyyknl96vg")
```

Comment:
Let's increase the lines removal count! We don't need that `USB_MUSB_PIO_ONLY` since it went unused in the end.

If it's needed, we can re-introduce it. It's not expected to be needed with the current project state.